rmgpy/molecule/molecule.py

```diff
@@ -135,7 +135,11 @@ def __reduce__(self):
             'atomType': self.atomType.label if self.atomType else None,
             'lonePairs': self.lonePairs,
         }
-        return (Atom, (self.element.symbol, self.radicalElectrons, self.charge, self.label), d)
+        if self.element.isotope == -1:
```

Comment:
Why do we not need additional changes in processing when we unpickle the isotopic element?

Reply:
When we unpickle the pickled file, it performs the following operations:

```
# create object of name Argument 0 using args from Argument 1
new_object = [Argument 0]([Argument 1])
# call custom_made method to set other arguments
new_object.__setstate__([Argument 2])
```

For the atom class could look something like 

```
new_object = Atom('C',0,0,'')
new_object.__setstate__(d)
```

Since the __init__ method for Atom already can properly accept either a string or an Element object, no changes in unpickling are necessary.

Reply:
Ok